js/checkout.js

```diff
@@ -301,7 +301,7 @@ class Blockonomics {
             }
         ).then(res => {
             this._update_order_params(res)
-            setTimeout( () => this._set_refresh_loading(false), 500)
```

Comment:
I added the delay to counter the effects of `reset_progress` (without the change that was done in this PR), so that when the counter resets, it gives us the same time to reset the animations as before the next `tick` is triggered